packages/admin-stories/src/docs/components/Toolbar/Toolbar.stories.mdx

```diff
@@ -0,0 +1,185 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs/blocks";
+
+<Meta title="Docs/Components/Toolbar" />
+
+# Toolbar
+
+The Toolbar System is a lightweight drop-in system to build a common and recognizable UI Pattern over all `@comet/admin` screens. It offers a structured place for Back Button, Title, Breadcrumbs, Action Buttons, Filters, Filter Bar, and many other UI Components which concern the currently presented screen.
+
+## Components
+
+The Toolbar system currently consists of some basic components. Everything is designed to offer developers a structured, easy usable, customizable and extendable toolbar system. From the User perspective the `@comet/admin` Toolbar should offer an application wide consistent UI, where the user consistently knows where one can find the back button, breadcrumbs, screen title or call to action buttons.
+
+Currently supported and built-in components:
+
+-   `Toolbar`
+-   `ToolbarBackButton`
+-   `ToolbarBreadcrumbs`
+-   `ToolbarAutomaticTitleItem`
+-   `ToolbarFillSpace`
+-   `ToolbarItem`
+-   `ToolbarActions`
+
+### Toolbar
+
+An empty Toolbar is nothing really fancy and not even more than a Mui Paper/ Mui Toolbar which sticks on the right place. Do not use empty toolbars at all, that makes really no sense. Try to always put a title, back button or at some rare cases breadcrumbs into it. Even on an entry screen with just a table, it is recommended to add a toolbar with a title. If the user moves to a detail screen, the toolbar sticks at the same place and changes the content according to the needs of the new screen.
```

Comment:
I would replace this section with the following paragraph:

The `Toolbar` component is the wrapper for all `Toolbar*` child components and takes care of the positioning of the toolbar.

> It is not recommended to use the `Toolbar` component without using at least one child `Toolbar*` component. If a screen doesn't require a dedicated action inside the toolbar, you can either add a title or remove the toolbar at all.